.env.test

```diff
@@ -0,0 +1,6 @@
+ DB_PASSWORD=password
```

Comment:
This file is in the `gitignore`, so it should not have been committed. Guessing it was added before the entry in gitignore was? Normally this would be a security risk in the wild having ever been committed, just something to be aware of going forward